aten/src/ATen/LegacyTHDispatch.cpp

```diff
@@ -0,0 +1,12 @@
+#include <ATen/LegacyTHDispatch.h>
+
+namespace at {
+
+// TODO: This could be bad juju if someone calls globalContext() in the
```

Comment:
Comment out of date

Reply:
is it?  This is the same comment as in globalLegacyTypeDispatch -- is it out of date there?  I'll admit I don't know what the comment is referring to.